qa/tasks/mgr/dashboard/test_mgr_module.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you please add the file to ```modules``` section in ```qa/suites/rados/mgr/tasks/dashboard.yaml``` otherwise it won't get tested by Teuthology ;)

Reply:
Thx for the tip.